xl_auth/user/models.py

```diff
@@ -158,6 +159,43 @@ def __repr__(self):
         return '<PasswordReset({email!r})>'.format(email=self.user.email)
 
 
+class LoginAttempt(SurrogatePK, Model):
```

Comment:
I'm so unhappy with this name! The only time it's clear what it represents is when reading this docstring below. Even though I know what it does now, my first reaction when I see it in the code is "someone attempted to login", then I correct myself with "no, yeah, it's *someone failed to login*".

It's better when things say what they mean;
* PasswordReset – a code that can reset someone's password
* User – a user
* Collection – a named bunch of "something"

Can we please include the word "failed" somewhere? **FailedLoginAttempt**, **FailedLogin**, **LoginFailure**, **UnsuccessfulAttemptToLogin**?

* LoginFailure – a failed login attempt, maybe curated upon successful login

Reply:
Fixed.